packages/@lwc/template-compiler/src/__tests__/codegen/codegen.spec.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import fs from 'fs';
+import path from 'path';
+import generate from '../../codegen';
+import parseTemplate from '../../parser';
+import { normalizeConfig } from '../../config';
+import State from '../../state';
+
+const TEST_DIRECTORIES = [
+    {
+        input: './fixtures/simple/input.html',
+        json: './fixtures/simple/input.json',
+        output: './fixtures/simple/expected.js',
+    },
+    {
+        input: './fixtures/if-else-if-else/input.html',
+        json: './fixtures/if-else-if-else/input.json',
+        output: './fixtures/if-else-if-else/expected.js',
+    },
+    {
+        input: './fixtures/if/input.html',
+        json: './fixtures/if/input.json',
+        output: './fixtures/if/expected.js',
+    },
+    {
+        input: './fixtures/if-flatten/input.html',
+        json: './fixtures/if-flatten/input.json',
+        output: './fixtures/if-flatten/expected.js',
+    },
+];
+
+const MODIFIED_AST_DIRECTORIES = [
+    {
+        input: './fixtures/for-if-simple/input.html',
+        json: './fixtures/for-if-simple/input.json',
+        output: './fixtures/for-if-simple/expected.js',
+    },
+];
+
+describe('codegen', () => {
```

Comment:
This directory is just for testing and verify, they will not be included in the final output